webpack/ForemanTasks/Components/TaskDetails/TaskDetails.js

```diff
@@ -76,13 +80,13 @@ class TaskDetails extends Component {
               taskProgressToggle={this.taskProgressToggle}
             />
           </Tab>
-          <Tab eventKey={3} title={__('Errors')}>
+          <Tab eventKey={3} disabled={loading} title={__('Errors')}>
             <Errors executionPlan={executionPlan} failedSteps={failedSteps} />
           </Tab>
-          <Tab eventKey={4} title={__('Locks')}>
+          <Tab eventKey={4} disabled={loading} title={__('Locks')}>
             <Locks locks={locks} />
           </Tab>
-          <Tab eventKey={5} title={__('Raw')}>
+          <Tab eventKey={5} disabled={loading} title={__('Raw')}>
```

Comment:
how about showing a tabs loading skeleton or a large spinner until the data is loaded,
and only then show the real tabs? feels like a better UX

Reply:
What do you mean by tabs loading skeleton? AFAIK we only have rows loading skeleton

Reply:
it's the "skeleton" of the page, we can mimic the way the page looks and give it a loading feeling instead of using a large spinner, e.g:
![1_yk5bG1DBaOU_KiXASE9dsA](https://user-images.githubusercontent.com/26363699/85405820-9db06c80-b569-11ea-83b4-8ea611ea24a5.gif)

Reply:
We don't have this implemented in PF or foreman, and adding a skeleton to each information line in the code will look messy. Also the spinner in the screenshot is the largest one we have

Reply:
we spoke about it offline back than, the library that we are using for `skeleton`
in foreman is [react-loading-skeleton](https://github.com/dvtng/react-loading-skeleton#react-loading-skeleton). with this library you can customize even full pages loading, instead of spinners.. which result in a better UX.

see also [foreman usage](https://github.com/theforeman/foreman/blob/7553eea533daa7de9f2f6f7ee1a827d15d3c714e/webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/components/AuditsLoadingPage.js#L2)

Reply:
![Screenshot from 2020-07-09 17-23-54](https://user-images.githubusercontent.com/30431079/87052018-fee26c00-c208-11ea-8f6d-29e32d446eda.png)
We decided to use one big skeleton, what do you think of this one?

Reply:
Does it have animation?

Reply:
yeah, the video quality is not the best :sweat_smile: 
![Screencast from 07-09-2020 05_20_05 PM](https://user-images.githubusercontent.com/30431079/87057971-2e48a700-c210-11ea-8826-edfc596cf105.gif)

Reply:
it feels a bit heavy as a one big piece,
I would suggest replacing the existing buttons/text with the current layout, it feels lighter:
![Untitled_ Jul 12, 2020 3_13 PM](https://user-images.githubusercontent.com/26363699/87245933-54f62000-c452-11ea-87bc-787e82c231c0.gif)

you can do the same for the rest of the tabs, and it will be a better UX than disabling/enabling each tab

Reply:
I don't think we should do it for each tab as the loading time won't be long enough for it to matter.. This loading state is only when the page opens. I'll add the split skeleton to this PR